### ISSUE TITLE
JS bitwise else if minifies incorrectly

### ISSUE THREAD

**Issue by JRaspass** [tdewolff/minify#335]
Apologies if this works with the unreleased code, I've only tested the current latest published version.

This
```javascript
function foo(bar) {
    if ((bar & 0xFFFFF800) == 0) {
        return 1;
    }
    else if ((bar & 0xFFFF0000) == 0) {
        return 2;
    }
    return 3;
}
```

Minifes to this, note the lack of `return 3` at all
```javascript
function foo(a){if((a&4294965248)==0)return 1;if((a&4294901760)==0)return 2;if((a&2492471136)==0)return 2}
```

**Comment by tdewolff** [comment]
Thanks for the bug report, I'll release a new version soon!

**Comment by JRaspass** [comment]
Thank you for a fast fix!

**Comment by tdewolff** [comment]
Has been released in v2.9.6!
